create.go

```diff
@@ -116,6 +130,42 @@ func getConfigs(bundlePath, containerID, console string) (vc.PodConfig, ShimConf
 	return *podConfig, shimConfig, *ociSpec, nil
 }
 
+func createCgroupsFiles(cgroupsPath string, pid int) error {
+	if cgroupsPath == "" {
+		return fmt.Errorf("Missing cgroups path")
+	}
+
+	cgroupsDir := filepath.Join(cgroupsMemDirPath, cgroupsPath)
```

Comment:
and make sure we understand if there is a difference between 'the spec' and what 'runc' does/supports. :-)